models.py

```diff
@@ -235,35 +235,93 @@ def update(self, data, current_user):
         self.put()
 
     @staticmethod
-    def bounding_box_query(ne_lat, ne_lng, sw_lat, sw_lng, start_date, end_date,
-                           fatal, severe, light, inaccurate, show_markers=True, is_thin=False, yield_per=None):
+    def bounding_box_query(is_thin=False, yield_per=None, **kwargs):
+
         # example:
         # ne_lat=32.36292402647484&ne_lng=35.08873443603511&sw_lat=32.29257266524761&sw_lng=34.88445739746089
         # >>>  m = Marker.bounding_box_query(32.36, 35.088, 32.292, 34.884)
         # >>> m.count()
         # 250
 
-        if not show_markers:
+        if not kwargs['show_markers']:
```

Comment:
Use `kwargs.get('show_markers', True)` here, since then if `show_markers` isn't given it will be treated as `True` rather than raising a `ValueError`. This is to preserve the optionality of `show_markers` as a parameter.